UpdateDB/DataLoading/AffixDataLoader.cs

```diff
@@ -233,13 +235,42 @@ private static IEnumerable<XmlStat> ExtractStats(string statColumn, string affix
                 }
                 stat = rangeRenameFunc(affix, stat);
                 var fromTo = stat.Split(new[] {" to "}, StringSplitOptions.None);
-                yield return new XmlStat
+                var from = fromTo[0].ParseFloat();
+                var to = fromTo.Length > 1 ? fromTo[1].ParseFloat() : from;
+                xmlStats.Add(new XmlStat
                 {
                     Name = affix,
-                    From = fromTo[0].ParseFloat(),
-                    To = fromTo.Length > 1 ? fromTo[1].ParseFloat() : fromTo[0].ParseFloat()
-                };
+                    From = new[] { from },
+                    To = new[] { to }
+                });
+            }
+
+            // merge "... # minimum ..., ... # maximum" into "... # to # ..."
+            if (xmlStats.Count > 1)
+            {
+                var previous = xmlStats[0];
+                for (var i = 1; i < xmlStats.Count; i++)
```

Comment:
Yeah, ReSharper's default settings suggest using var for everything. It's only marked as "Hint" though (not as a code smell).

It's probably just me being lazy in most cases. I should definitely change my habit of using var everywhere and use only use it where the type is obvious. Changing int to var in loops doesn't really make sense at all, it doesn't even save space.